model/task/db.go

```diff
@@ -666,6 +666,25 @@ func FindOneIdWithFields(id string, projected ...string) (*Task, error) {
 	return task, nil
 }
 
+func FindManyWithFields(query db.Q, projected ...string) ([]Task, error) {
```

Comment:
can we have this be FindAll, to match existing terminology.

also can we consolidate logic, so that we have a function that's GetTaskIDs(v version) (or versionID that returns a slice of strings. 

In short: wrap database operations *entirely* in public model functions. (don't pass queries to them, don't project all the data out, and make callers post-process.) Make them really boring

Reply:
done